<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowClass</class>
 <widget class="QMainWindow" name="mainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinLepp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/main_icon/logo.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="mainPanel" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="modePanel" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="consoleButton">
            <property name="toolTip">
             <string>Inicia la UI en modo consola (menos recursos)</string>
            </property>
            <property name="text">
             <string>Consola</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/mode/console.png</normaloff>:/mode/console.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uiButton">
            <property name="toolTip">
             <string>Inicia la UI y dibuja los triangulos en pantalla (más recursos)</string>
            </property>
            <property name="text">
             <string>UI</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/mode/meshscene.png</normaloff>:/mode/meshscene.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="uiPanel" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Seleccionar</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="clearSelectionButton">
               <property name="toolTip">
                <string>Limpiar Selección</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../resources/resources.qrc">
                 <normaloff>:/selection/select_none.png</normaloff>:/selection/select_none.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="selectAllButton">
               <property name="toolTip">
                <string>Seleccionar Todos</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../resources/resources.qrc">
                 <normaloff>:/selection/select_all.png</normaloff>:/selection/select_all.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="selectAngleButton">
               <property name="toolTip">
                <string>Slección basada en ángulos</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../resources/resources.qrc">
                 <normaloff>:/selection/select_angle.png</normaloff>:/selection/select_angle.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Opciones</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Zoom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sceneScaleCombo">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>10%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>25%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>50%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>100%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>200%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>400%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1000%</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="consolePanel" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Algoritmo de Selección</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="selectionCriterionComboBox">
               <item>
                <property name="text">
                 <string>Seleccionar todos</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección basada en ángulos</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección Aleatoria (50%)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección Aleatoria (10%)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección Aleatoria (5%)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección Aleatoria (3%)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Selección Aleatoria (1%)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="refineButton">
         <property name="text">
          <string>Refinar</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resources.qrc">
           <normaloff>:/main_icon/logo.png</normaloff>:/main_icon/logo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionCargar_Mesh"/>
    <addaction name="actionGuardar_Malla"/>
    <addaction name="separator"/>
    <addaction name="action_Salir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionConcurrencia"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionCargar_Mesh">
   <property name="text">
    <string>Cargar Malla</string>
   </property>
  </action>
  <action name="actionGuardar_Malla">
   <property name="text">
    <string>Guardar Malla</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionCriterios_de_Selecci_n">
   <property name="text">
    <string>Criterios de Selección</string>
   </property>
  </action>
  <action name="actionConcurrencia">
   <property name="text">
    <string>Concurrencia</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Salir</sender>
   <signal>triggered()</signal>
   <receiver>mainwindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
